roles/oneagent/tasks/provide-installer/download-unix.yml

```
@@ -19,3 +20,7 @@
   no_log: "{{ oneagent_no_log }}"
   ignore_errors: true
   register: _oneagent_download_result
+  retries: 5
+  until: _oneagent_download_result is not failed
+  delay: 10
+  changed_when: false
```

This makes it so changed status is not reported when the installer is downloaded.
I think this suggestion would make it report the appropriate status:
```suggestion
  changed_when: not installer_stat.stat.exists or _oneagent_download_result.changed
```

---

done